chart/templates/globalrole.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: management.cattle.io/v3
+builtin: false
+description: "Elemental Administrator Role"
+displayName: Elemental Administrator
+kind: GlobalRole
+metadata:
+  finalizers:
+  - controller.cattle.io/mgmt-auth-gr-controller
+  labels:
```

Comment:
I'm not sure if we want the label or finalizer. It might be worth asking what the impact of these fields are?

Reply:
I removed finalizer, but I would keep label, because it might be not visible in UI.